tests/src/util/playgrounds/unique.dev.ts

```diff
@@ -84,7 +85,32 @@ class ArrangeGroup {
       }
     }
 
-    await Promise.all(transactions);
+    await Promise.all(transactions).catch(e => {});
+    
+    //#region TODO remove this region, when nonce problem will be solved
+    const checkBalances = async () => {
+      let isSuccess = true;
+      for (let i = 0; i < balances.length; i++) {
+        const balance = await this.helper.balance.getSubstrate(accounts[i].address);
+        if (balance !== balances[i] * tokenNominal) {
+          isSuccess = false;
+          break;
+        }
+      }
+      return isSuccess;
+    };
+
+    let accountsCreated = false;
+    // waiting up to 1 minute with .25 sec retry
+    for (let index = 0; index < 240; index++) {
```

Comment:
Maybe we should use blocks there instead of seconds?
240 iterations of 0.25 s = 60 seconds = 5 blocks
So, we can use `helper.chain.getLatestBlockNumber` and check balances only when it changes.
In that case we call our rpc only `5 * balances.length` times, instead of `240 * balances.length`